roles/ee_builder/defaults/main.yml

```diff
@@ -1,24 +1,26 @@
 ---
 # build_arg_defaults
-builder_dir: "{{ playbook_dir }}"
 ee_container_runtime: podman
 ee_version: 1
 galaxy_cli_opts: '-v'
-# ee_base_image: quay.io/ansible/ansible-runner:devel
-# builder_image:
 
-# Req File defaults
-bindep_file: bindep
```

Comment:
We've gone back a step now (sorry I initially caused the confusion here. There were still used and I think you've made it so they now aren't used but it makes sense someone MAY want to override it so it makes sense still being there. Could you add them back in. I'll then flag where they should go back

Reply:
I've reverted, I think there was something around trying to hard code it to make things easier, but you are right, someone may want to overwrite it.